<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Azael Avalos &lt;coproscefalo@gmail.com&gt;</author>
 <class>HDDProtect</class>
 <widget class="QWidget" name="HDDProtect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HDD Protect</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>500</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="hdd_protect_label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>HDD Protection:</string>
         </property>
         <property name="buddy">
          <cstring>hdd_protect_checkbox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="hdd_protect_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="hdd_protect_checkbox">
         <property name="toolTip">
          <string>Toggles the HDD protection monitoring.</string>
         </property>
         <property name="whatsThis">
          <string>Toggles the HDD protection monitoring.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="hdd_notification_label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>HDD Protection Notification:</string>
         </property>
         <property name="buddy">
          <cstring>hdd_notification_checkbox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="hdd_notification_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="hdd_notification_checkbox">
         <property name="toolTip">
          <string>Notify if the HDD monitoring detected movement.</string>
         </property>
         <property name="whatsThis">
          <string>Notify if the HDD monitoring detected movement.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="protection_level_label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>HDD Protection Level:</string>
         </property>
         <property name="buddy">
          <cstring>protection_level_slider</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="protection_level">
         <property name="toolTip">
          <string>The current HDD protection level.</string>
         </property>
         <property name="whatsThis">
          <string>The current HDD protection level.</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QSlider" name="protection_level_slider">
         <property name="toolTip">
          <string>Sets the HDD protection level.</string>
         </property>
         <property name="whatsThis">
          <string>Sets the HDD protection level.</string>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
